src/components/calcite-action-bar/calcite-action-bar.tsx

```diff
@@ -119,15 +125,20 @@ export class CalciteActionBar {
   // --------------------------------------------------------------------------
 
   renderBottomActionGroup() {
-    const { expanded, expand, textExpand, textCollapse, el, layout, toggleExpand } = this;
+    const { expanded, expand, textExpand, textCollapse, el, layout, position, toggleExpand } = this;
+    const positionFallback: CalcitePosition = layout
```

Comment:
Nitpick: `position` should have higher precedence in the odd case both are provided.